beacon-chain/sync/pending_blocks_queue_test.go

```diff
@@ -192,3 +199,83 @@ func TestRegularSyncBeaconBlockSubscriber_ProcessPendingBlocks2(t *testing.T) {
 		t.Errorf("Incorrect size for seen pending block: got %d", len(r.seenPendingBlocks))
 	}
 }
+
+func TestRegularSyncBeaconBlockSubscriber_PruneOldPendingBlocks(t *testing.T) {
+	db := dbtest.SetupDB(t)
+	defer dbtest.TeardownDB(t, db)
+	p1 := p2ptest.NewTestP2P(t)
+	p2 := p2ptest.NewTestP2P(t)
+	p1.Connect(p2)
+	if len(p1.Host.Network().Peers()) != 1 {
+		t.Error("Expected peers to be connected")
+	}
+	pcl := protocol.ID("/eth2/beacon_chain/req/hello/1/ssz")
```

Comment:
What does hello handshake have to do with this test?

Reply:
alright done, removed